nss-day-cohort-46/reactive-nutshell-ring-of-sunfire-1
# Corrected code for edit button

# Description
Edit button is working with no errors 
It ended up being a route syntax error in the provider
Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Check and see if edit is working

git checkout main
git fetch --all
git checkout tw-NewStuff



# Checklist:
